frontend/src/config/i18n/de/guidedTour.json

```
@@ -8,6 +8,8 @@
     "confirmation_pause": "Pausieren",
     "confirmation_quit": "Beenden",
     "fetch_status_error": "Sorry, ich habe Schwierigkeiten den Status deiner Einführung in den Map Editor zu empfangen. Bitte lade die Seite neu, damit ich es noch einmal versuchen kann.",
+    "update_tour_status_error": "Sorry, ich habe Schwierigkeiten den Status deiner Einführung in den Map Editor zu aktualisieren.",
```

Could you actually trigger this error? Is reloading of the page maybe useful?

```suggestion
    "update_tour_status_error": "Entschuldigung, ich habe Schwierigkeiten den Status deiner Einführung zu aktualisieren.",
```